record-editor.py

```diff
@@ -813,22 +834,26 @@ def create_models( self ):
         #
         # NOTE: we keep art id in the model so we can pull it from our
         #       selection and access the record's data.
-        self.artModel = QStandardItemModel( 0, 2, self )
+        self.artModel = QStandardItemModel( 0, self.NUM_COLUMNS, self )
 
-        self.artModel.setHeaderData( 0, Qt.Horizontal, "Art ID" )
-        self.artModel.setHeaderData( 1, Qt.Horizontal, "State" )
+        self.artModel.setHeaderData( self.ID_COLUMN, Qt.Horizontal, "Art ID" )
+        self.artModel.setHeaderData( self.STATE_COLUMN, Qt.Horizontal, "State" )
 
         # walk through each of the photo records and insert a new item at the
         # beginning of the model's list.
         for art in self.art_records:
-            self.artModel.insertRow( 0 )
-            self.artModel.setData( self.artModel.index( 0, 0 ), art["id"] )
-            self.artModel.setData( self.artModel.index( 0, 1 ), art["state"] )
+            id_item = QStandardItem( str( art["id"] ) )
+            state_item = QStandardItem( art["state"] )
+
+            id_item.setData( art["id"] )
+            state_item.setData( art["state"] )
+
+            self.artModel.appendRow( [id_item, state_item] )
```

Comment:
Whatever happens with the PhotoRecordViewer's photoModel above should be done here as well.  There won't ever be enough records to worry about performance though consistency is key.